apps/climatemappedafrica/src/components/HURUmap/TreeView/TreeView.js

```diff
@@ -0,0 +1,205 @@
+import { Box, SvgIcon } from "@mui/material";
+import { styled } from "@mui/material/styles";
+import { SimpleTreeView } from "@mui/x-tree-view/SimpleTreeView";
+import {
+  TreeItem2Checkbox,
+  TreeItem2Content,
+  TreeItem2GroupTransition,
+  TreeItem2IconContainer,
+  TreeItem2Label,
+  TreeItem2Root,
+} from "@mui/x-tree-view/TreeItem2";
+import { TreeItem2Icon } from "@mui/x-tree-view/TreeItem2Icon";
+import { TreeItem2Provider } from "@mui/x-tree-view/TreeItem2Provider";
+import { useTreeItem2 } from "@mui/x-tree-view/useTreeItem2";
+import clsx from "clsx";
+import PropTypes from "prop-types";
+import React from "react";
+
+import CheckIcon from "@/climatemappedafrica/assets/icons/checked.svg";
+import slugify from "@/climatemappedafrica/utils/slugify";
+
+const CustomTreeItemContent = styled(TreeItem2Content)(({ theme }) => ({
+  background: "inherit",
+  borderRadius: 0,
+  borderBottom: `1px solid transparent`,
+  borderRight: `2px solid transparent`,
+  padding: 0,
+  "&:hover": {
+    background: "inherit",
+  },
```

Comment:
I think we don't really need this right as its the same style ?

Reply:
This is needed. Without it, 👇🏽 

https://github.com/user-attachments/assets/cebcf516-3347-4d60-a003-8ccb7679c45f

Reply:
Anha, I see 🚀